### ISSUE TITLE
custom_ui broken with HA 0.77.2

### ISSUE THREAD

**Issue by edif30** [andrey-git/home-assistant-custom-ui#152]
### Checklist
- [x ] I'm running the latest version of CustomUI ([Update guide](https://github.com/andrey-git/home-assistant-custom-ui/blob/master/docs/installing.md#updating)) or using a specific [release](https://github.com/andrey-git/home-assistant-custom-ui/releases) that is not marked as "Broken".
- [x ] I tried to force-refresh (Ctrl+Shift+R / Ctrl+F5) the browser
- [x ] (Optional, but recommended) I'm using Chrome or tried to reproduce the feature on Chrome.

**Browser + Version:** latest version

**CustomUI version:** latest

**Home Assistant release (`hass --version`):** 0.77.2

**Problem-relevant `configuration.yaml` entries:**

**Problem-relevant Home Assistant log entries:**
Per HA devs... 

For custom card developers that were relying on hass.config, it has been split up. hass.config.core is now available as hass.config and hass.config.services is now available as hass.services

This breaks custom_ui completely in 0.77.2

**Comment by fanaticDavid** [comment]
Ah, I believe I'm seeing this too.

Just today, I happened to upgrade my Home Assistant install from 0.71.x to 0.77.2. After setting up the user account, I get to see the Home Assistant UI but the cards are pretty much blank. They also keep on increasing in size, as a little popup in the lower left corner notifies me an error has been written to the Home Assistant log file. I've had to completely strip Custom UI from my configuration for the Home Assistant UI to be usable.

That's one way to see just how much Custom UI improves the Home Assistant experience...
